cmake/SetupCatch.cmake

```diff
@@ -4,10 +4,10 @@
 find_package(Catch 2.8.0 REQUIRED)
 
 spectre_include_directories("${CATCH_INCLUDE_DIR}")
-message(STATUS "Catch include: ${CATCH_INCLUDE_DIR}")
-message(STATUS "Catch version: ${CATCH_VERSION}")
+message(STATUS "Catch incl: ${CATCH_INCLUDE_DIR}")
+message(STATUS "Catch vers: ${CATCH_VERSION}")
```

Comment:
Why abbreviate? Better write "version" I think. (Also for Boost)

Reply:
For consistency with all our other SetupBlah cmake output 🙂... I would be just as happy to make everything output the unabbreviated text instead (vers/incl/libs -> version/include/library), but that's a bigger diff

Reply:
I think some of your diff in this PR uses `version`, so I suggest to just go with that. No need to refactor all occurrences in this PR.

Reply:
🤷 okay, reverting this change